prisma/seed.ts

```diff
@@ -1,25 +1,64 @@
-import { Addon, PrismaClient, StageType } from "@prisma/client";
-
-function createCards(addon: Addon) {
-  const cards = [];
-  const range = Math.floor(Math.random() * (300 - 30 + 1)) + 30;
-  for (let index = 0; index < range; index++) {
-    cards.push({
-      addon_id: addon.id,
-      is_nsfw: Math.random() < 0.5,
-      stages: [
-        {
-          title: `card_${addon.title}_${index}`,
-          text: "test",
-          type: StageType.TEXT,
-        },
-      ],
-    });
-  }
-  return cards;
-}
+import { Addon, Game, PrismaClient, StageType } from "@prisma/client";
+import { faker } from "@faker-js/faker";
 
 const prisma = new PrismaClient();
+const createArray = (min: number, max: number) => faker.datatype.array(faker.datatype.number({ min, max }));
+
+function createTextStage() {
+  const a = faker.lorem.sentence()
+    .split(" ")
+    .map(x => Math.random() < 0.1
+      ? faker.helpers.arrayElement(["%PLAYER1%", "%SELF%", "%TURNS%"])
+      : x
+    );
+
+  const hasTurns = a.indexOf("%TURNS%") !== -1;
+  const text = a.join(" ");
+
+  return {
+    type: StageType.TEXT,
+    title: faker.random.words(5),
+    text,
+    turns: hasTurns ? faker.datatype.number(12) : undefined,
```

Comment:
`faker.datatype.number` defaults to a minimum of 0, we probably don't want that

```suggestion
    turns: hasTurns ? faker.datatype.number({ min: 1, max: 12 }) : undefined,
```